<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (1.8.0_292) on Thu Oct 06 18:45:03 EDT 2022 -->
<title>AlignmentSummaryMetrics (picard 2.27.5-SNAPSHOT API)</title>
<meta name="date" content="2022-10-06">
<link rel="stylesheet" type="text/css" href="../../stylesheet.css" title="Style">
<script type="text/javascript" src="../../script.js"></script>
</head>
<body>
<script type="text/javascript"><!--
    try {
        if (location.href.indexOf('is-external=true') == -1) {
            parent.document.title="AlignmentSummaryMetrics (picard 2.27.5-SNAPSHOT API)";
        }
    }
    catch(err) {
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar.top">
<!--   -->
</a>
<div class="skipNav"><a href="#skip.navbar.top" title="Skip navigation links">Skip navigation links</a></div>
<a name="navbar.top.firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../index-all.html">Index</a></li>
<li><a href="../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../picard/analysis/AdapterUtility.html" title="class in picard.analysis"><span class="typeNameLink">Prev&nbsp;Class</span></a></li>
<li><a href="../../picard/analysis/AlignmentSummaryMetrics.Category.html" title="enum in picard.analysis"><span class="typeNameLink">Next&nbsp;Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../index.html?picard/analysis/AlignmentSummaryMetrics.html" target="_top">Frames</a></li>
<li><a href="AlignmentSummaryMetrics.html" target="_top">No&nbsp;Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../allclasses-noframe.html">All&nbsp;Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li><a href="#nested.class.summary">Nested</a>&nbsp;|&nbsp;</li>
<li><a href="#field.summary">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor.summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#methods.inherited.from.class.htsjdk.samtools.metrics.MetricBase">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li><a href="#field.detail">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor.detail">Constr</a>&nbsp;|&nbsp;</li>
<li>Method</li>
</ul>
</div>
<a name="skip.navbar.top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<!-- ======== START OF CLASS DATA ======== -->
<div class="header">
<div class="subTitle">picard.analysis</div>
<h2 title="Class AlignmentSummaryMetrics" class="title">Class AlignmentSummaryMetrics</h2>
</div>
<div class="contentContainer">
<ul class="inheritance">
<li>java.lang.Object</li>
<li>
<ul class="inheritance">
<li>htsjdk.samtools.metrics.MetricBase</li>
<li>
<ul class="inheritance">
<li><a href="../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">picard.metrics.MultilevelMetrics</a></li>
<li>
<ul class="inheritance">
<li>picard.analysis.AlignmentSummaryMetrics</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<div class="description">
<ul class="blockList">
<li class="blockList">
<hr>
<br>
<pre>@DocumentedFeature(groupName="Metrics",
                   groupSummary="Metrics",
                   summary="Alignment metrics")
public class <span class="typeNameLabel">AlignmentSummaryMetrics</span>
extends <a href="../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</a></pre>
<div class="block">High level metrics about the alignment of reads within a SAM file, produced by
 the CollectAlignmentSummaryMetrics program and usually stored in a file with
 the extension ".alignment_summary_metrics".</div>
</li>
</ul>
</div>
<div class="summary">
<ul class="blockList">
<li class="blockList">
<!-- ======== NESTED CLASS SUMMARY ======== -->
<ul class="blockList">
<li class="blockList"><a name="nested.class.summary">
<!--   -->
</a>
<h3>Nested Class Summary</h3>
<table class="memberSummary" border="0" cellpadding="3" cellspacing="0" summary="Nested Class Summary table, listing nested classes, and an explanation">
<caption><span>Nested Classes</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static class&nbsp;</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.Category.html" title="enum in picard.analysis">AlignmentSummaryMetrics.Category</a></span></code>&nbsp;</td>
</tr>
</table>
</li>
</ul>
<!-- =========== FIELD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="field.summary">
<!--   -->
</a>
<h3>Field Summary</h3>
<table class="memberSummary" border="0" cellpadding="3" cellspacing="0" summary="Field Summary table, listing fields, and an explanation">
<caption><span>Fields</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Field and Description</th>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#AVG_POS_3PRIME_SOFTCLIP_LENGTH">AVG_POS_3PRIME_SOFTCLIP_LENGTH</a></span></code>
<div class="block">The average length of the soft-clipped bases at the 3' end of reads.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#BAD_CYCLES">BAD_CYCLES</a></span></code>
<div class="block">The number of instrument cycles in which 80% or more of base calls were no-calls.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../picard/analysis/AlignmentSummaryMetrics.Category.html" title="enum in picard.analysis">AlignmentSummaryMetrics.Category</a></code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#CATEGORY">CATEGORY</a></span></code>
<div class="block">One of either UNPAIRED (for a fragment run), FIRST_OF_PAIR when metrics are for only the
 first read in a paired run, SECOND_OF_PAIR when the metrics are for only the second read
 in a paired run or PAIR when the metrics are aggregated for both first and second reads
 in a pair.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#MAD_READ_LENGTH">MAD_READ_LENGTH</a></span></code>
<div class="block">The median absolute deviation of the distribution of all read lengths.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#MAX_READ_LENGTH">MAX_READ_LENGTH</a></span></code>
<div class="block">The maximum read length.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#MEAN_ALIGNED_READ_LENGTH">MEAN_ALIGNED_READ_LENGTH</a></span></code>
<div class="block">The mean aligned read length of the set of reads examined.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#MEAN_READ_LENGTH">MEAN_READ_LENGTH</a></span></code>
<div class="block">The mean read length of the set of reads examined.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#MEDIAN_READ_LENGTH">MEDIAN_READ_LENGTH</a></span></code>
<div class="block">The median read length of the set of reads examined.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#MIN_READ_LENGTH">MIN_READ_LENGTH</a></span></code>
<div class="block">The minimum read length.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PCT_ADAPTER">PCT_ADAPTER</a></span></code>
<div class="block">The fraction of PF reads that are unaligned or aligned with MQ0 and match to a known adapter sequence right from the
 start of the read (indication of adapter-dimer pairs).</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PCT_CHIMERAS">PCT_CHIMERAS</a></span></code>
<div class="block">The fraction of reads that map outside of a maximum insert size (usually 100kb) or that have
 the two ends mapping to different chromosomes.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PCT_HARDCLIP">PCT_HARDCLIP</a></span></code>
<div class="block">The fraction of PF bases that are (on primary, aligned reads and) hard-clipped, as a fraction of the
 PF_ALIGNED_BASES (even though these are not aligned!)</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PCT_PF_READS">PCT_PF_READS</a></span></code>
<div class="block">The fraction of reads that are PF (PF_READS / TOTAL_READS)</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PCT_PF_READS_ALIGNED">PCT_PF_READS_ALIGNED</a></span></code>
<div class="block">The percentage of PF reads that aligned to the reference sequence.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PCT_PF_READS_IMPROPER_PAIRS">PCT_PF_READS_IMPROPER_PAIRS</a></span></code>
<div class="block">The fraction of (primary) reads that are *not* "properly" aligned in pairs (as per SAM flag 0x2).</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PCT_READS_ALIGNED_IN_PAIRS">PCT_READS_ALIGNED_IN_PAIRS</a></span></code>
<div class="block">The fraction of aligned reads whose mate pair was also aligned to the reference.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PCT_SOFTCLIP">PCT_SOFTCLIP</a></span></code>
<div class="block">the fraction of PF bases that are on (primary) aligned reads and are soft-clipped, as a fraction of the
 PF_ALIGNED_BASES (even though these are not aligned!)</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PF_ALIGNED_BASES">PF_ALIGNED_BASES</a></span></code>
<div class="block">The total number of aligned bases, in all mapped PF reads, that are aligned to the reference sequence.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_ALIGNED_BASES">PF_HQ_ALIGNED_BASES</a></span></code>
<div class="block">The number of bases aligned to the reference sequence in reads that were mapped at high
 quality.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_ALIGNED_Q20_BASES">PF_HQ_ALIGNED_Q20_BASES</a></span></code>
<div class="block">The subset of PF_HQ_ALIGNED_BASES where the base call quality was Q20 or higher.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_ALIGNED_READS">PF_HQ_ALIGNED_READS</a></span></code>
<div class="block">The number of PF reads that were aligned to the reference sequence with a mapping quality of
 Q20 or higher signifying that the aligner estimates a 1/100 (or smaller) chance that the
 alignment is wrong.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_ERROR_RATE">PF_HQ_ERROR_RATE</a></span></code>
<div class="block">The fraction of bases that mismatch the reference in PF HQ aligned reads.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PF_HQ_MEDIAN_MISMATCHES">PF_HQ_MEDIAN_MISMATCHES</a></span></code>
<div class="block">The median number of mismatches versus the reference sequence in reads that were aligned
 to the reference at high quality (i.e.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PF_INDEL_RATE">PF_INDEL_RATE</a></span></code>
<div class="block">The number of insertion and deletion events per 100 aligned bases.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PF_MISMATCH_RATE">PF_MISMATCH_RATE</a></span></code>
<div class="block">The rate of bases mismatching the reference for all bases aligned to the reference sequence.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PF_NOISE_READS">PF_NOISE_READS</a></span></code>
<div class="block">The number of PF reads that are marked as noise reads.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PF_READS">PF_READS</a></span></code>
<div class="block">The number of PF reads where PF is defined as passing Illumina's filter.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PF_READS_ALIGNED">PF_READS_ALIGNED</a></span></code>
<div class="block">The number of PF reads that were aligned to the reference sequence.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#PF_READS_IMPROPER_PAIRS">PF_READS_IMPROPER_PAIRS</a></span></code>
<div class="block">The number of (primary) aligned reads that are **not** "properly" aligned in pairs (as per SAM flag 0x2).</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#READS_ALIGNED_IN_PAIRS">READS_ALIGNED_IN_PAIRS</a></span></code>
<div class="block">The number of aligned reads whose mate pair was also aligned to the reference.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#SD_READ_LENGTH">SD_READ_LENGTH</a></span></code>
<div class="block">The standard deviation of the read lengths.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>double</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#STRAND_BALANCE">STRAND_BALANCE</a></span></code>
<div class="block">The number of PF reads aligned to the positive strand of the genome divided by the number of
 PF reads aligned to the genome.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#TOTAL_READS">TOTAL_READS</a></span></code>
<div class="block">The total number of reads including all PF and non-PF reads.</div>
</td>
</tr>
</table>
<ul class="blockList">
<li class="blockList"><a name="fields.inherited.from.class.picard.metrics.MultilevelMetrics">
<!--   -->
</a>
<h3>Fields inherited from class&nbsp;picard.metrics.<a href="../../picard/metrics/MultilevelMetrics.html" title="class in picard.metrics">MultilevelMetrics</a></h3>
<code><a href="../../picard/metrics/MultilevelMetrics.html#LIBRARY">LIBRARY</a>, <a href="../../picard/metrics/MultilevelMetrics.html#READ_GROUP">READ_GROUP</a>, <a href="../../picard/metrics/MultilevelMetrics.html#SAMPLE">SAMPLE</a></code></li>
</ul>
</li>
</ul>
<!-- ======== CONSTRUCTOR SUMMARY ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor.summary">
<!--   -->
</a>
<h3>Constructor Summary</h3>
<table class="memberSummary" border="0" cellpadding="3" cellspacing="0" summary="Constructor Summary table, listing constructors, and an explanation">
<caption><span>Constructors</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tr class="altColor">
<td class="colOne"><code><span class="memberNameLink"><a href="../../picard/analysis/AlignmentSummaryMetrics.html#AlignmentSummaryMetrics--">AlignmentSummaryMetrics</a></span>()</code>&nbsp;</td>
</tr>
</table>
</li>
</ul>
<!-- ========== METHOD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="method.summary">
<!--   -->
</a>
<h3>Method Summary</h3>
<ul class="blockList">
<li class="blockList"><a name="methods.inherited.from.class.htsjdk.samtools.metrics.MetricBase">
<!--   -->
</a>
<h3>Methods inherited from class&nbsp;htsjdk.samtools.metrics.MetricBase</h3>
<code>equals, hashCode, toString</code></li>
</ul>
<ul class="blockList">
<li class="blockList"><a name="methods.inherited.from.class.java.lang.Object">
<!--   -->
</a>
<h3>Methods inherited from class&nbsp;java.lang.Object</h3>
<code>clone, finalize, getClass, notify, notifyAll, wait, wait, wait</code></li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
<div class="details">
<ul class="blockList">
<li class="blockList">
<!-- ============ FIELD DETAIL =========== -->
<ul class="blockList">
<li class="blockList"><a name="field.detail">
<!--   -->
</a>
<h3>Field Detail</h3>
<a name="CATEGORY">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>CATEGORY</h4>
<pre>public&nbsp;<a href="../../picard/analysis/AlignmentSummaryMetrics.Category.html" title="enum in picard.analysis">AlignmentSummaryMetrics.Category</a> CATEGORY</pre>
<div class="block">One of either UNPAIRED (for a fragment run), FIRST_OF_PAIR when metrics are for only the
 first read in a paired run, SECOND_OF_PAIR when the metrics are for only the second read
 in a paired run or PAIR when the metrics are aggregated for both first and second reads
 in a pair.</div>
</li>
</ul>
<a name="TOTAL_READS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>TOTAL_READS</h4>
<pre>public&nbsp;long TOTAL_READS</pre>
<div class="block">The total number of reads including all PF and non-PF reads. When CATEGORY equals PAIR
 this value will be 2x the number of clusters.</div>
</li>
</ul>
<a name="PF_READS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_READS</h4>
<pre>public&nbsp;long PF_READS</pre>
<div class="block">The number of PF reads where PF is defined as passing Illumina's filter.</div>
</li>
</ul>
<a name="PCT_PF_READS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_PF_READS</h4>
<pre>public&nbsp;double PCT_PF_READS</pre>
<div class="block">The fraction of reads that are PF (PF_READS / TOTAL_READS)</div>
</li>
</ul>
<a name="PF_NOISE_READS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_NOISE_READS</h4>
<pre>public&nbsp;long PF_NOISE_READS</pre>
<div class="block">The number of PF reads that are marked as noise reads.  A noise read is one which is composed
 entirely of A bases and/or N bases. These reads are marked as they are usually artifactual and
 are of no use in downstream analysis.</div>
</li>
</ul>
<a name="PF_READS_ALIGNED">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_READS_ALIGNED</h4>
<pre>public&nbsp;long PF_READS_ALIGNED</pre>
<div class="block">The number of PF reads that were aligned to the reference sequence. This includes reads that
 aligned with low quality (i.e. their alignments are ambiguous).</div>
</li>
</ul>
<a name="PCT_PF_READS_ALIGNED">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_PF_READS_ALIGNED</h4>
<pre>public&nbsp;double PCT_PF_READS_ALIGNED</pre>
<div class="block">The percentage of PF reads that aligned to the reference sequence. PF_READS_ALIGNED / PF_READS</div>
</li>
</ul>
<a name="PF_ALIGNED_BASES">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_ALIGNED_BASES</h4>
<pre>public&nbsp;long PF_ALIGNED_BASES</pre>
<div class="block">The total number of aligned bases, in all mapped PF reads, that are aligned to the reference sequence.</div>
</li>
</ul>
<a name="PF_HQ_ALIGNED_READS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_HQ_ALIGNED_READS</h4>
<pre>public&nbsp;long PF_HQ_ALIGNED_READS</pre>
<div class="block">The number of PF reads that were aligned to the reference sequence with a mapping quality of
 Q20 or higher signifying that the aligner estimates a 1/100 (or smaller) chance that the
 alignment is wrong.</div>
</li>
</ul>
<a name="PF_HQ_ALIGNED_BASES">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_HQ_ALIGNED_BASES</h4>
<pre>public&nbsp;long PF_HQ_ALIGNED_BASES</pre>
<div class="block">The number of bases aligned to the reference sequence in reads that were mapped at high
 quality.  Will usually approximate PF_HQ_ALIGNED_READS * READ_LENGTH but may differ when
 either mixed read lengths are present or many reads are aligned with gaps.</div>
</li>
</ul>
<a name="PF_HQ_ALIGNED_Q20_BASES">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_HQ_ALIGNED_Q20_BASES</h4>
<pre>public&nbsp;long PF_HQ_ALIGNED_Q20_BASES</pre>
<div class="block">The subset of PF_HQ_ALIGNED_BASES where the base call quality was Q20 or higher.</div>
</li>
</ul>
<a name="PF_HQ_MEDIAN_MISMATCHES">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_HQ_MEDIAN_MISMATCHES</h4>
<pre>public&nbsp;double PF_HQ_MEDIAN_MISMATCHES</pre>
<div class="block">The median number of mismatches versus the reference sequence in reads that were aligned
 to the reference at high quality (i.e. PF_HQ_ALIGNED READS).</div>
</li>
</ul>
<a name="PF_MISMATCH_RATE">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_MISMATCH_RATE</h4>
<pre>public&nbsp;double PF_MISMATCH_RATE</pre>
<div class="block">The rate of bases mismatching the reference for all bases aligned to the reference sequence.</div>
</li>
</ul>
<a name="PF_HQ_ERROR_RATE">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_HQ_ERROR_RATE</h4>
<pre>public&nbsp;double PF_HQ_ERROR_RATE</pre>
<div class="block">The fraction of bases that mismatch the reference in PF HQ aligned reads.</div>
</li>
</ul>
<a name="PF_INDEL_RATE">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_INDEL_RATE</h4>
<pre>public&nbsp;double PF_INDEL_RATE</pre>
<div class="block">The number of insertion and deletion events per 100 aligned bases.  Uses the number of events
 as the numerator, not the number of inserted or deleted bases.</div>
</li>
</ul>
<a name="MEAN_READ_LENGTH">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>MEAN_READ_LENGTH</h4>
<pre>public&nbsp;double MEAN_READ_LENGTH</pre>
<div class="block">The mean read length of the set of reads examined.  When looking at the data for a single lane with
 equal length reads this number is just the read length.  When looking at data for merged lanes with
 differing read lengths this is the mean read length of all reads. Computed using all read lengths
 including clipped bases.</div>
</li>
</ul>
<a name="SD_READ_LENGTH">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>SD_READ_LENGTH</h4>
<pre>public&nbsp;double SD_READ_LENGTH</pre>
<div class="block">The standard deviation of the read lengths. Computed using all read lengths including clipped bases.</div>
</li>
</ul>
<a name="MEDIAN_READ_LENGTH">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>MEDIAN_READ_LENGTH</h4>
<pre>public&nbsp;double MEDIAN_READ_LENGTH</pre>
<div class="block">The median read length of the set of reads examined.  When looking at the data for a single lane with
 equal length reads this number is just the read length.  When looking at data for merged lanes with
 differing read lengths this is the median read length of all reads. Computed using all bases in reads,
 including clipped bases.</div>
</li>
</ul>
<a name="MAD_READ_LENGTH">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>MAD_READ_LENGTH</h4>
<pre>public&nbsp;double MAD_READ_LENGTH</pre>
<div class="block">The median absolute deviation of the distribution of all read lengths.  If the distribution is
 essentially normal then the standard deviation can be estimated as ~1.4826 * MAD. Computed using all
 read lengths including clipped bases.</div>
</li>
</ul>
<a name="MIN_READ_LENGTH">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>MIN_READ_LENGTH</h4>
<pre>public&nbsp;double MIN_READ_LENGTH</pre>
<div class="block">The minimum read length. Computed using all read lengths including clipped bases.</div>
</li>
</ul>
<a name="MAX_READ_LENGTH">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>MAX_READ_LENGTH</h4>
<pre>public&nbsp;double MAX_READ_LENGTH</pre>
<div class="block">The maximum read length. Computed using all read lengths including clipped bases.</div>
</li>
</ul>
<a name="MEAN_ALIGNED_READ_LENGTH">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>MEAN_ALIGNED_READ_LENGTH</h4>
<pre>public&nbsp;double MEAN_ALIGNED_READ_LENGTH</pre>
<div class="block">The mean aligned read length of the set of reads examined.  When looking at the data for a single lane with
 equal length reads this number is just the read length.  When looking at data for merged lanes with
 differing read lengths this is the mean read length of all reads. Clipped bases are not counted.</div>
</li>
</ul>
<a name="READS_ALIGNED_IN_PAIRS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>READS_ALIGNED_IN_PAIRS</h4>
<pre>public&nbsp;long READS_ALIGNED_IN_PAIRS</pre>
<div class="block">The number of aligned reads whose mate pair was also aligned to the reference.</div>
</li>
</ul>
<a name="PCT_READS_ALIGNED_IN_PAIRS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_READS_ALIGNED_IN_PAIRS</h4>
<pre>public&nbsp;double PCT_READS_ALIGNED_IN_PAIRS</pre>
<div class="block">The fraction of aligned reads whose mate pair was also aligned to the reference.
 READS_ALIGNED_IN_PAIRS / PF_READS_ALIGNED</div>
</li>
</ul>
<a name="PF_READS_IMPROPER_PAIRS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PF_READS_IMPROPER_PAIRS</h4>
<pre>public&nbsp;long PF_READS_IMPROPER_PAIRS</pre>
<div class="block">The number of (primary) aligned reads that are **not** "properly" aligned in pairs (as per SAM flag 0x2).</div>
</li>
</ul>
<a name="PCT_PF_READS_IMPROPER_PAIRS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_PF_READS_IMPROPER_PAIRS</h4>
<pre>public&nbsp;double PCT_PF_READS_IMPROPER_PAIRS</pre>
<div class="block">The fraction of (primary) reads that are *not* "properly" aligned in pairs (as per SAM flag 0x2).
 PF_READS_IMPROPER_PAIRS / PF_READS_ALIGNED</div>
</li>
</ul>
<a name="BAD_CYCLES">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>BAD_CYCLES</h4>
<pre>public&nbsp;long BAD_CYCLES</pre>
<div class="block">The number of instrument cycles in which 80% or more of base calls were no-calls.</div>
</li>
</ul>
<a name="STRAND_BALANCE">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>STRAND_BALANCE</h4>
<pre>public&nbsp;double STRAND_BALANCE</pre>
<div class="block">The number of PF reads aligned to the positive strand of the genome divided by the number of
 PF reads aligned to the genome.</div>
</li>
</ul>
<a name="PCT_CHIMERAS">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_CHIMERAS</h4>
<pre>public&nbsp;double PCT_CHIMERAS</pre>
<div class="block">The fraction of reads that map outside of a maximum insert size (usually 100kb) or that have
 the two ends mapping to different chromosomes.</div>
</li>
</ul>
<a name="PCT_ADAPTER">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_ADAPTER</h4>
<pre>public&nbsp;double PCT_ADAPTER</pre>
<div class="block">The fraction of PF reads that are unaligned or aligned with MQ0 and match to a known adapter sequence right from the
 start of the read (indication of adapter-dimer pairs).</div>
</li>
</ul>
<a name="PCT_SOFTCLIP">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_SOFTCLIP</h4>
<pre>public&nbsp;double PCT_SOFTCLIP</pre>
<div class="block">the fraction of PF bases that are on (primary) aligned reads and are soft-clipped, as a fraction of the
 PF_ALIGNED_BASES (even though these are not aligned!)</div>
</li>
</ul>
<a name="PCT_HARDCLIP">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>PCT_HARDCLIP</h4>
<pre>public&nbsp;double PCT_HARDCLIP</pre>
<div class="block">The fraction of PF bases that are (on primary, aligned reads and) hard-clipped, as a fraction of the
 PF_ALIGNED_BASES (even though these are not aligned!)</div>
</li>
</ul>
<a name="AVG_POS_3PRIME_SOFTCLIP_LENGTH">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>AVG_POS_3PRIME_SOFTCLIP_LENGTH</h4>
<pre>public&nbsp;double AVG_POS_3PRIME_SOFTCLIP_LENGTH</pre>
<div class="block">The average length of the soft-clipped bases at the 3' end of reads. This could be used as an estimate for
 the amount by which the insert-size must be increased in order to obtain a significant reduction in bases
 lost due to reading off the end of the insert.</div>
</li>
</ul>
</li>
</ul>
<!-- ========= CONSTRUCTOR DETAIL ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor.detail">
<!--   -->
</a>
<h3>Constructor Detail</h3>
<a name="AlignmentSummaryMetrics--">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>AlignmentSummaryMetrics</h4>
<pre>public&nbsp;AlignmentSummaryMetrics()</pre>
</li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
</div>
<!-- ========= END OF CLASS DATA ========= -->
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar.bottom">
<!--   -->
</a>
<div class="skipNav"><a href="#skip.navbar.bottom" title="Skip navigation links">Skip navigation links</a></div>
<a name="navbar.bottom.firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../index-all.html">Index</a></li>
<li><a href="../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../picard/analysis/AdapterUtility.html" title="class in picard.analysis"><span class="typeNameLink">Prev&nbsp;Class</span></a></li>
<li><a href="../../picard/analysis/AlignmentSummaryMetrics.Category.html" title="enum in picard.analysis"><span class="typeNameLink">Next&nbsp;Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../index.html?picard/analysis/AlignmentSummaryMetrics.html" target="_top">Frames</a></li>
<li><a href="AlignmentSummaryMetrics.html" target="_top">No&nbsp;Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../allclasses-noframe.html">All&nbsp;Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li><a href="#nested.class.summary">Nested</a>&nbsp;|&nbsp;</li>
<li><a href="#field.summary">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor.summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#methods.inherited.from.class.htsjdk.samtools.metrics.MetricBase">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li><a href="#field.detail">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor.detail">Constr</a>&nbsp;|&nbsp;</li>
<li>Method</li>
</ul>
</div>
<a name="skip.navbar.bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
</body>
</html>
